wordpress-mobile/WordPress-iOS
# People: Removes Followers + Viewers Fullname

Fixes #5691

To test:
1. Log into a dotcom account
2. Open a blog in which you're an admin (**My Sites** >> **Testing Site**)
3. Open the `People` section

Please, verify that the **Followers** + **Viewers** don't display **First Name** + **Last Name** anymore.

Note that **Viewers** is only available for private sites.

Needs review: @kurzee  
Thanks in advance Brent!!
